7.js

```diff
@@ -2,24 +2,38 @@
 
 // Q 7 : What is the 10001st prime number?
 
-// function to find prime number array 
-
-
-
-function findPrimeByIndex(num){
-    let primes = [];
-    let count = 0;
+// function to check if a given number is a prime number 
+
+function checkPrime(checkNum){
+    let isPrime= true;
+    for (let i=2;i<checkNum;i++){
+        if(checkNum%i===0 && i!==checkNum){
+            isPrime = false;
+        }
+    }
+    return isPrime
+}
 
-    while(count<num){
+// function to find prime number array 
 
-     
-    count++;
+function findNPrime(num){
+    let primeCount = 0;
+    let iterateNum = 2;
+    let primeNum = 0;
+
+    while(primeCount<num){
+        
+        if(checkPrime(iterateNum)){
+            primeCount++
+            primeNum = iterateNum
+        } 
+        iterateNum++
     }
+    
+    return primeNum
 
-   return primes;
 }
 
 // main code 
 
-console.log(findPrimeByIndex(10001))
```

Comment:
It might be a good idea to also log how much time this function takes to run. :)

You can use this to do it:
https://developer.mozilla.org/en-US/docs/Web/API/console/timelog_static

Reply:
Also, that way, as you come up with better approaches, you can see how you improve

Reply:
thank you, i will try it!